runtime/src/lib.rs

```diff
@@ -257,10 +255,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("sora-substrate"),
     impl_name: create_runtime_str!("sora-substrate"),
     authoring_version: 1,
-    spec_version: 100,
+    spec_version: 101,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
-    transaction_version: 100,
+    transaction_version: 101,
```

Comment:
Do you want  to deploy it on Stage without new version? Like 4.2.1

Reply:
Yes, because for now we just want to release it on stage and time for release on mainnet is not defined yet